src/components/BreedInfo.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import {useSelector} from "react-redux";
+import { makeStyles } from '@material-ui/core';
+import { Paper } from '@material-ui/core';
+import StarIcon from '@material-ui/icons/Star';
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        display: "flex",
+        flexDirection: "column",
+        alignItems: "center",
+    },
+    images: {
+        display: "flex",
+        justifyContent: "center",
+        flexWrap: "wrap",
+    },
+    imgContainer: {
+        minWidth: 200,
+        maxHeight: 300,
+    },
+    img: {
+        maxHeight: 300,
+    },
+    paper: {
+        display: 'flex',
+        textAlign: "center",
+        flexWrap: 'wrap',
+        '& > *': {
+            margin: theme.spacing(1),
+            width: 'auto',
+            height: '100%',
+            backgroundColor: '#C3CBD6',
+            marginBottom: '50px'
+        },
+    },
+}));
+
+export const BreedInfo = () => {
+    const classes = useStyles();
+//
+    const state = useSelector(store=> store);
+    const clickedBreed = useSelector(store=>store.clickedBreed)
+    const selectedBreedObjFromStore = state.breedsSlice.breeds.filter((obj)=> obj.id === clickedBreed);
```

Comment:
Redundant all store getting on 42 line, it is possible to just 
`const selectedBreedObjFromStore = useSelector(store => store.breedsSlice.breeds.find((obj)=> obj.id === clickedBreed)`